### PR TITLE
MINOR: Bump version to 2.3.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by mjsax**

### CODE REVIEW

File: tests/kafkatest/version.py [hunk 1]
```diff
@@ -117,3 +117,7 @@ def get_version(node=None):
 # 2.1.x versions
 V_2_1_0 = KafkaVersion("2.1.0")
 LATEST_2_1 = V_2_1_0
+
+# 2.2.x versions
+V_2_2_0 = KafkaVersion("2.2.0")
+LATEST_2_2 = V_2_2_0
```
Comment (ijuma) [inline tests/kafkatest/version.py]: I think you should only do this when the version actually is published.

**Review by ijuma — APPROVED** [review]
Just a comment, looks good otherwise.

**Comment by mjsax** [comment]
Updated.

**Review by ijuma — APPROVED** [review]
LGTM

**Comment by mjsax** [comment]
https://builds.apache.org/job/kafka-pr-jdk11-scala2.12/2132/
```
java.lang.AssertionError: Did not get valid initial assignment for partitions [topic2-3, topic1-2, topic2-0, topic2-5, topic1-4, topic2-7, topic2-1, topic2-6, topic1-0, topic1-3, topic2-4, topic1-1, topic2-2]
	at kafka.utils.TestUtils$.fail(TestUtils.scala:356)
	at kafka.utils.TestUtils$.waitUntilTrue(TestUtils.scala:766)
	at kafka.api.PlaintextConsumerTest.validateGroupAssignment(PlaintextConsumerTest.scala:1772)
	at kafka.api.PlaintextConsumerTest.testMultiConsumerRoundRobinAssignment(PlaintextConsumerTest.scala:864)
```

Retest this please.

**Pull request merged**